targets/unity-v2/source/ExampleTestProject/Assets/PlayFabEditorExtensions/Editor/Scripts/Utils/PlayFabEditorHelper.cs

```diff
@@ -22,6 +22,10 @@ public static partial class PlayFabEditorHelper
         public static string EDEX_UPGRADE_PATH = "/Resources/PlayFabUnityEditorExtensions.unitypackage";
         public static string EDEX_PACKAGES_PATH = "/Resources/MostRecentPackage.unitypackage";
 
+        public static string AAD_SIGNIN_URL = "https://login.microsoftonline.com/";
+        public static string ED_EX_AAD_SIGNIN_CLIENTID ="448adbda-b8d8-4f33-a1b0-ac58cf44d4c1";
```

Comment:
I think we should be using 2d99511e-13ec-4b59-99c0-9ae8754f84aa as the client id. The 448... id is just used in the requested scope and represents the audience for which we're requesting the login token.